### DISCUSSION TITLE
Is it possible to add several directories?

### DISCUSSION THREAD

**Discussion by ParsaArvanehPA** [EnzDev/idea-ngx-translate-autocomplete#80] [Q&A] [answered]
Hi,
I was wondering is it possible to add several directories containing json files?
The project I'm working on is made of more than 10 application, and each have their own i18n directory.

**Comment by EnzDev** [comment]
Hello, I'll be reworking on the plugin soon, will add a better configuration support where you can add custom directories easily. 
If you don't specify a directory in the configuration, the plugin looks for JSON files under any folder named `assets` in the project (at any depth so `src/assets/myI18n/file.json`, `app/assets/lang.json` would work).

**Comment by EnzDev** [comment, marked as answer]
Hello,  I have been really busy at work and has 0 time to work on the plugin...
Currently, if you do not set up a directory for the translations (first parameter) the application will recursively search for '.json' files into directories that are marked as 'Resource Directory'.
You can either set multiple directories as Resource directory or a common parent if you do not have other json files.

![resource directory marked](https://user-images.githubusercontent.com/14039086/209006456-da0f4ecb-d034-48f4-8c4a-f07d223ee0fa.png)
![settings](https://user-images.githubusercontent.com/14039086/209006347-0a291453-287e-4bc4-a200-6b5d65130469.png)

**Reply by ParsaArvanehPA** [reply]
Thanks for your response.
